jxl/src/frame/modular/predict.rs

```diff
@@ -152,17 +155,156 @@ impl Predictor {
     }
 }
 
+const NUM_PREDICTORS: usize = 4;
+const PRED_EXTRA_BITS: i64 = 3;
+const PREDICTION_ROUND: i64 = ((1 << PRED_EXTRA_BITS) >> 1) - 1;
+const DIVLOOKUP: [u32; 64] = [
```

Comment:
Please description on how this magic constants are obtained (even better, calculate them in compile time).

Original comment was `for (int i = 0; i < 64; i++) divlookup[i] = (1 << 24) / (i + 1);`

Reply:
I have no idea why they are what they are - I just copied them from libjxl.

Added a comment explaining that.

Reply:
But I have no idea how the constants are obtained, all I can do is refer to the libjxl codebase.

Reply:
Oh, you mean copy the comment from libjxl about the DIVLOOKUP constant in particular. Sure, done.